app/models/user.rb

```diff
@@ -3,7 +3,8 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
-  has_many :metrics
+  
+  has_many :metrics, through: :features
```

Comment:
I was lucky enough to find an example of exactly what I wanted to do in guides.rubyonrails.org, [it's at the end of this section:](http://guides.rubyonrails.org/association_basics.html#the-has-many-through-association)

```
class Document < ActiveRecord::Base
  has_many :sections
  has_many :paragraphs, through: :sections
end
 
class Section < ActiveRecord::Base
  belongs_to :document
  has_many :paragraphs
end
 
class Paragraph < ActiveRecord::Base
  belongs_to :section
end
```